services/bios/genesis.json

```diff
@@ -1,6 +1,6 @@
 {
   "initial_timestamp": "2020-08-11T04:20:00.000",
-  "initial_key": "EOS7GY6pQ86iWNS3DtS15gQ96ZNoyoNr9Ca7Y74mqUDmWunmbz9DP",
```

Comment:
@JustinCast  can we keep the original key? let me know and i can send you the private key for it